tests/framework-tests/core/js/DataBindingTests.js

```diff
@@ -56,6 +56,120 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             }
         });
     };
+
+    fluid.tarzanTests = [{
```

Comment:
Did you mean <code>tarjanTests</code>, or is this a joke?

Reply:
I am sorry to report that this is a form of ioke.